src/datepicker/datepicker.component.ts

```diff
@@ -203,13 +208,25 @@ export class DatePicker implements OnDestroy, OnChanges, AfterViewChecked {
 		}
 	}
 
+	ngAfterViewInit() {
+		const scrollObservable = scrollableParentsObservable(this.elementRef.nativeElement);
+		this.scrollSubscription = scrollObservable.subscribe((event: any) => {
```

Comment:
Just so it doesn't get lost, this should use `ElementService` 👍

Reply:
Working on that now 👍

Reply:
Awesome 💯